specification/dimensions/billingcurrency.md

```diff
@@ -0,0 +1,76 @@
+### Billing Currency
```

Comment:
start with H1 and go to H2/3 as needed. You'll find examples in other files / PRs